floris/tools/floris_interface_legacy_reader.py

```diff
@@ -123,7 +123,7 @@ def _convert_v24_dictionary_to_v3(dict_legacy):
     dict_floris["flow_field"] = {
         "air_density": fp["air_density"],
         "reference_wind_height": ref_height,
-        "turbulence_intensity": fp["turbulence_intensity"][0],
+        "turbulence_intensities": fp["turbulence_intensities"][0],
```

Comment:
I don't think this should be changed; this function is called `_convert_v24_dictionary_to_v3`, so it would be confusing if it didn't take as an input a valid v2 model and return a valid v3 model. However, perhaps the entire floris_interface_legacy_reader.py module is going away?